activemq/publisherAMQP.go

```diff
@@ -0,0 +1,74 @@
+package activemq
+
+import (
+	"context"
+	"github.com/andile-innovation/popcorn/log"
+	"net"
+	"pack.ag/amqp"
+	"time"
+)
+
+type NewAMQPPublisherRequest struct {
+	Host string
+	Port int
+}
+
+func NewAMQPPublisher(request NewAMQPPublisherRequest) AMQPPublisher {
+	return AMQPPublisher{
+
+	}
+}
+
+type AMQPPublisher struct {
+	Client *amqp.Client
+	Conn net.Conn
+}
+
+func (p *AMQPPublisher) Connect(address string) error {
```

Comment:
Don't pass address to connect - rater utilise constructor parameters `p.host` and `p.port`

Reply:
@Hennie97 did you resolver this?

Reply:
I resolved it and pushed it now.